robottelo/hosts.py

```diff
@@ -551,7 +551,7 @@ def create_custom_repos(self, **kwargs):
 
         For example::
 
-            create_custom_repo(custom_repo='http://repourl.domain.com/path')
+            create_custom_repo(custom_repo='http://repourl.domain.com/path', proxy='http://proxy.domain.com:1234')
```

Comment:
The implementation of this method doesn't appear to match the implementation. This example would result in a file `/etc/yum.repos.d/proxy.repo`:
```ini
[proxy]
name=proxy
baseurl=http://proxy.domain.com:1234
enabled=1
gpgcheck=0
```